user-interface/src/lib/components/uswds/DateRangePicker.tsx

```diff
@@ -27,6 +27,7 @@ function DateRangePickerComponent(props: DateRangePickerProps, ref: React.Ref<In
   const [startDateValue, setStartDateValue] = useState<string | undefined>(undefined);
   const [endDateValue, setEndDateValue] = useState<string | undefined>(undefined);
 
+  // set timeout added to solve some funky weirdness with the date type input handling keyboard events after a reset.
```

Comment:
Is this like a debounce issue? If so, maybe we could define a debounce function so it's clear why we're doing the setTimeout.